packages/git/src/browser/git-navigable-list-widget.ts

```diff
@@ -52,12 +54,10 @@ export class GitNavigableListWidget<T extends { selected?: boolean }> extends Vi
 
     protected onUpdateRequest(msg: Message): void {
         super.onUpdateRequest(msg);
-
         (async () => {
             const selected = this.node.getElementsByClassName(SELECTED_CLASS)[0];
-            const scrollArea = await this.getScrollContainer();
-            if (selected && scrollArea) {
-                ElementExt.scrollIntoViewIfNeeded(scrollArea, selected);
+            if (selected && this.scrollArea) {
```

Comment:
`this.scrollArea` does not have  `undefined` in type declaration, why is it checked?